Api/Report/BaseReport.php

```diff
@@ -35,8 +37,18 @@ public function setReturnOnlyCompleteData($returnOnlyCompleteData)
         return $this;
     }
 
-    public function getReportLanguage($language)
+    public function setReportLanguage($language)
     {
         $this->reportRequest->Language = $language;
     }
+
+    protected function createReportRequest()
+    {
+        $this->reportRequest = new ReportRequest();
```

Comment:
If this is only like thie we can just do it in the constructor, or is it overwritten by child classes?